backend/src/app/services/disclosure/disclosure.service.ts

```diff
@@ -35,7 +35,7 @@ export class DisclosureService {
 
       if (showPending) {
         result = await this.disclosureRepository.find({
-          where: { siteId, userAction: UserActionEnum.UPDATED },
+          where: { siteId, srAction: SRApprovalStatusEnum.PENDING },
```

Comment:
any related changes required for the unit test ?

Reply:
No unit test were failing due to this.